Documentation/ApiOverview/SoftReferences/Index.rst

```diff
@@ -205,11 +205,89 @@ The content could look like this:
    <p>To read more about <a href="http://example.org/some-cool-feature">this cool feature</a></p>
    <p>Contact: email@example.org</p>
 
-The parsers will return an array containing information about
-
-#. A page link
-#. An external link
-#. An email link
+The parsers will return an array containing information about the references
+contained in the string::
+
+   [
+       'content' => '
+          <p><a href="{424242}">Congratulations</a></p>
+          <p>To read more about <a href="{softref:78910}">this cool feature</a></p>
+          <p>Contact: {softref:123456}</p>
+       ',
+       'elements' => [
+           [
+               'matchString' => '<a href="t3://page?uid=96">',
+               'error' => 'There is a glitch in the universe, page 42 not found.',
+               'subst' => [
+                   'type' => 'db','424242',
```

Comment:
This looks odd and would lead to an `0 => '424242'` array entry ...

Reply:
fixed it, key was missing, maybe I accidently deleted something